### ISSUE TITLE
Mask is an array full of 0, leads to Nan for logits

### ISSUE THREAD

**Issue by gabrielvannier** [clks-wzz/FAS-SGTD#22]
Thanks for this project !
This project is awesome. However I have an issue that occur really often and I can not figure out how to solve it :/

I am testing the repository in multi_frame using your script test.py. When starting the test, everything works fine and the predict fonction yields features correctly except for one thing :  For almost half of the features the mask is an array full of 0 for the first two dimensions.

This imply that the new depth map (calculated on line 151 of test.py will be an array full of 0 :
`depth_map = depth_map[..., 0]*masks[..., 0]  `

and the depth_mean will be an Nan value (because of division by 0) : 
`depth_mean = np.sum(depth_map) / np.sum(masks[..., 0])`

And logits as well...
`cla_ratio = flags.paras.cla_ratio
  depth_ratio = 1 - cla_ratio
  logits[0] = depth_ratio * depth_mean + cla_ratio * logits[0]
  logits[1] = 1.0 - depth_mean`

So I need to figure out why the mask is an array full of 0, and how to fix the it to get the output I want ! 
If anybody could help me I would be very thankful !

PS :  If you need any additional information let me know, I am 100% available

**Comment by gabrielvannier** [comment]
Also, what does the masks is supposed to represent ? what is its meaning ?

**Comment by nguyennhatkhoa99** [comment]
How do you calculate the depth map for input to the network. My image prediction always return to 0.
